src/main.cpp

```diff
@@ -140,17 +141,16 @@ auto main( int argc, char* argv[] ) -> int {
 		excludes.emplace_back( "sdk_content.*" );
 		excludes.emplace_back( ".*.vmf_autosave" );
 		excludes.emplace_back( ".*index.rsv" );
```

Comment:
Should the dots in extensions be escaped?

Reply:
doesn't really matter, one could, but a single char can't do anything bad